torch/_inductor/compile_fx.py

```diff
@@ -931,6 +931,18 @@ def log_graph_runnable() -> str:
                         print_output=False, include_stride=True, include_device=True
                     ),
                 )
+                if config.trace.enabled:
```

Comment:
There was a discussion on deprecating `TORCH_COMPILE_DEBUG`. Ok for now, but we may want to switch to TORCH_LOGS in future.